org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/ScanRequestCreationTest.java

```diff
@@ -193,7 +193,7 @@ public void testStepCommandWithMonitors() throws Exception {
 		pi.exec("sr =                               "
 			+	"scan_request(                      "
 			+	"    step(my_scannable, -2, 5, 0.5),"
-			+	"    mon=['x', another_scannable],  "  // Monitor two scannables.
+			+	"    monitorsPerPoint=['x', another_scannable],  "  // Monitor two scannables.
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "    monitorsPerPoint=['x', another_scannable],  " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)